api-reference/docstubs/api/user-delta.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "delta"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# delta
```

Comment:
Our current [topic title guidance](https://msgo.azurewebsites.net/add/document/guidelines/toc-and-topic-title.html) is to identify the type of the binding parameter and the method name, like in this case, "user: delta".

Reply:
Yup - already caught this one

Reply:
dupe